source/common/ssl/context_impl.h

```diff
@@ -37,8 +36,8 @@ struct SslStats {
   ALL_SSL_STATS(GENERATE_COUNTER_STRUCT, GENERATE_GAUGE_STRUCT, GENERATE_TIMER_STRUCT)
 };
 
-typedef CSmartPtr<SSL, SSL_free> SslConPtr;
-typedef CSmartPtr<X509, X509_free> X509Ptr;
+typedef bssl::UniquePtr<SSL> SslConPtr;
+typedef bssl::UniquePtr<X509> X509Ptr;
```

Comment:
You don't have to type alias, but when you do, you should follow the conventions in the style guide. To me, it depends on how much typing/verbosity vs. the clarity in a specific case. Here I think you can just skip.